static/styles/portico/portico-signin.scss

```diff
@@ -827,14 +836,14 @@ button#register_auth_button_gitlab {
         }
     }
 
+    #realm_with_subdomain ~ .realm_subdomain_label {
+        margin-left: 220px !important;
```

Comment:
What's the story behind this margin?

Reply:
I've increased the `margin-left` from 180px (line 831) to 220px in case of root domain availability to cope up with the increased width of radio-button input and word `Use`.

Reply:
We shouldn't use absolute positioning, if we don't have to. (I frankly don't know why `#registration .input-box label` specifies absolute positioning ...).

Reply:
Updated.

I've overridden the absolute positioning of `label` for subdomain input in both cases (root domain available and root domain not available).